packages/react-components/react-swatch-picker-preview/src/components/ColorSwatch/useColorSwatchStyles.styles.ts

```diff
@@ -43,6 +49,12 @@ const useStyles = makeResetStyles({
     outline: 'none',
     boxShadow: `inset 0 0 0 ${tokens.strokeWidthThick} ${tokens.colorStrokeFocus2}, inset 0 0 0 ${tokens.strokeWidthThicker} ${tokens.colorStrokeFocus1}`,
   }),
+  ':disabled': {
```

Comment:
Let's apply this based on props instead i.e. 

```ts
makeStyles({
  disabled: {
   ':hover': {
      cursor: 'not-allowed',
      boxShadow: 'none',
    },
  }
})

//

mergeClasses(
  // ...,
  state.disabled && styles.disabled,
)
```

Reply:
fixed